champions/src/main/java/me/mykindos/betterpvp/champions/champions/builds/menus/buttons/ClassSelectionButton.java

```diff
@@ -17,16 +20,20 @@
 import org.jetbrains.annotations.NotNull;
 import xyz.xenondevs.invui.item.impl.SimpleItem;
 
+import java.util.List;
+
 public class ClassSelectionButton extends SimpleItem {
 
     private final BuildManager buildManager;
     private final Role role;
     private final SkillManager skillManager;
     private final Windowed parent;
 
-    public ClassSelectionButton(BuildManager buildManager, SkillManager skillManager, Role role, Windowed parent) {
+    public ClassSelectionButton(BuildManager buildManager, SkillManager skillManager, Role role, ArmourManager armorManager, Windowed parent) {
         super(ItemView.builder().material(role.getChestplate())
                 .displayName(Component.text(role.getName(), role.getColor(), TextDecoration.BOLD))
+                .lore(List.of(UtilMessage.deserialize("Class Damage Reduction: <yellow>" + armorManager.getReductionForArmourSet(role.getChestplate().name().replace("_CHESTPLATE", "")) + "%"),
+                              UtilMessage.deserialize("Effective Health: <red>" + UtilFormat.formatNumber(20 * ( 1 + (armorManager.getReductionForArmourSet(role.getChestplate().name().replace("_CHESTPLATE", ""))/100)), 1))))
```

Comment:
true calc is: 20 / (1 - armorManager.getReductionForArmourSet(role.getChestplate().name().replace("_CHESTPLATE", "")) / 100)